tests/framework/pkg/namegenerator/namegenerator.go

```diff
@@ -0,0 +1,34 @@
+package namegenerator
+
+import (
+	"math/rand"
+	"time"
+)
+
+const lowerLetterBytes = "abcdefghijklmnopqrstuvwxyz"
+const upperLetterBytes = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const numberBytes = "0123456789"
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func RandStringLowerBytes(n int) string {
```

Comment:
Bytes makes  it sound like the function returns bytes.

```suggestion
func RandStringLower(n int) string {
```

Reply:
fixed